packages/vite/src/node/optimizer/scan.ts

```diff
@@ -486,6 +486,21 @@ function esbuildScanPlugin(
         }
       }
 
+      const globImportOnLoadCallback: (
+        args: OnLoadArgs,
+      ) => Promise<OnLoadResult | null | undefined> = async ({ path: p }) => {
+        const externalLoadResult: OnLoadResult = {
+          loader: 'js',
+          contents: '\nexport default {}',
+        }
+        if (!entries.includes(p)) {
+          return externalLoadResult
+        }
+        if (SPECIAL_QUERY_RE.test(p)) {
+          return externalLoadResult
+        }
```

Comment:
```suggestion
        return externalLoadResult
```
Why is this not like this?

Reply:
According to the original processing logic, not all paths need to execute `external` operations in the `onResolve` hook. In order to minimize modification of the logic, the logic here remains consistent with the original one.

Reply:
Ah, I see.